pom.xml

```diff
@@ -246,7 +246,7 @@
         <dependency>
             <groupId>org.eclipse.jdt</groupId>
             <artifactId>org.eclipse.jdt.annotation</artifactId>
-            <version>1.1.400</version>
+            <version>2.2.300</version>
```

Comment:
It's a majör version bump and it contains breaking changes that break all the annotation usage. Eclipse annotations moved to type annnotations (java 8) in version 2.x but Skript's code is NOT written in java 8 / type annotations in mind.

Upgrading the annotations version is just a hassle and in the end of day we have no gain over this because 1.x versions are also supported and get bug fixes etc. So upgrading is not a good idea.